### ISSUE TITLE
Change to use Scalar context when calling Text::Lorem

### ISSUE THREAD

**Issue by renatocron** [dagolden/Data-Fake#13]
Since Text::Lorem 0.32 it started checking if the caller want an array instead of scalar, see 
https://github.com/adeolaawoyemi/Text-Lorem/commit/1b079376247b47ecb4fe426f4fbd89875bb91d32#diff-a85f65f7acc0f94461c625c7df83ec79fb9d23df15e3bf84707d0a6346f353a3 

This broke a lot of previously working code, that was written like this:

```
{
                key1            => $ca_session,
                name               => 'foobar',
                description        => fake_sentences(2)->(),
                key_int          => 1,
                field3       => fake_int(1, 10_000)->(),
            }
```

Now, because of the fake_sentences(2) it returns 3 itens and make this hash non-balanced and including random keys!
For now I'm replacing every instance of fake_XXX with scalar() but I think this can cause some issue to other people as well

do you even intended to be used like this?

**Comment by xdg** [comment]
Thanks for letting me know!

**Comment by xdg** [comment]
Released to CPAN as Data-Fake-0.006
